ui/app/ducks/swaps/swaps.js

```diff
@@ -578,6 +587,33 @@ export const fetchQuotesAndSetQuoteState = (
   };
 };
 
+const isContractAddressValid = (
```

Comment:
[REQ] Could we add a jsdoc block to this function to describe its purpose briefly and expected parameters?

Reply:
Also because this doesn't interact with state or dispatch directly I think its better suited to be in the swaps utils file in the page folder. It can be directly exported from there instead of the testtables object

Reply:
Done.